bootstrap.yml

```diff
@@ -16,8 +16,18 @@
       - name: load instackenv file
         include_tasks: tasks/load_instackenv.yml
 
+      - name: check if the infrared dir exists
+        stat:
+          path: "{{ item }}"
+        register: infrared_dirs
+        loop:
+          - "{{ infrared_dir }}"
+          - "{{ infrared_workspaces_dir }}"
+
       - name: setup infrared
         include_tasks: tasks/setup_infrared.yml
+        when: item.stat.exists == False
```

Comment:
So if both ~/infrared and ~/.infrared doesn't exist, we are calling setup_infrared.yml twice, right?